File: Orquestra.Domain/Enums/UserRoleEnum.cs
using System.ComponentModel;

namespace Orquestra.Domain.Enums;

public enum UserRoleEnum
{
    [Description("Usuário do sistema")]
    Common = 1,

    [Description("Suporte do sistema")]
    Maintainer = 999,

    [Description("Administrador do sistema")]
    Admin = 1000
}

File: Orquestra.Frontend/app/enums/userRoleEnum.ts
export enum USER_ROLE_ENUM {
    Common = 1,
    Maintainer = 999,
    Administrator = 1000
}

export function getUserRoleDescription(role: USER_ROLE_ENUM) {
    switch (role) {
        case USER_ROLE_ENUM.Common:
            return 'Usuário do sistema';
        case USER_ROLE_ENUM.Maintainer:
            return 'Suporte do sistema';
        case USER_ROLE_ENUM.Administrator:
            return 'Administrador do sistema';
        default:
            return 'Desconhecido';
    }
}